docs/source/pages/howToWorkWithPlatform/adminApplication/callbacks/callbacks.rst

```diff
@@ -0,0 +1,94 @@
+===========
+Callbacks
```

Comment:
please make "=" signs to be in the line with title, it's necessary to fit sphynx markup

Reply:
done